cmd/sriovdp/manager.go

```diff
@@ -16,64 +16,114 @@ package main
 
 import (
 	"encoding/json"
+	"flag"
 	"fmt"
+	"io"
 	"os"
+	"path/filepath"
 
 	"github.com/golang/glog"
 	"github.com/jaypipes/ghw"
+	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/cdi"
 
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/factory"
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/types"
 	"github.com/k8snetworkplumbingwg/sriov-network-device-plugin/pkg/utils"
 )
 
 const (
-	socketSuffix = "sock"
+	defaultConfig = "/etc/pcidp/config.json"
+	socketSuffix  = "sock"
 )
 
-type cliParams struct {
-	configFile     string
-	resourcePrefix string
+// CliParams presents CLI parameters for SR-IOV Network Device Plugin
+type CliParams struct {
```

Comment:
Is there any particular reason to make all symbols in this file public?

It's not a big issue, but having those private allows changing signatures in a safer manner

Reply:
I need to revert this change. It will be required for a follow-up PR